components/amara/sources/common.mjs

```diff
@@ -0,0 +1,106 @@
+import amara from "../amara.app.mjs";
+import constants from "../constants.mjs";
+
+export default {
+  props: {
+    amara,
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      label: "Polling schedule",
+      description: "How often to poll the Amara API",
+      default: {
+        intervalSeconds: 60 * 15,
+      },
+    },
+    team: {
+      description: "The team slug of the team to watch",
+      optional: true,
```

Comment:
Yes, the idea here is that if you set `team` as either `null` or empty it will return resources in the public area. I've changed the description to be clearer.

Reply:
When I leave `team` blank in a source's configuration, I get an error when the source runs. E.g. for **Video Deleted**, I get the error:
```

{
  "status": 404,
  "statusText": "Not Found",
  "data": {
    "error": "not-found",
    "path": "/api/teams//activity/"
  }
}
```